### ISSUE TITLE
Questions about the paper

### ISSUE THREAD

**Issue by RegisWu** [yingcong/SemanticComponent#2]
Hi, it's a pleasure to learn your nice work!

I have a few questions about the original papers as follows:

1. May I know which loss functions did you use?

2. In the paper you mentioned you said k in Eq.(5) to 9 and then can disable insignificant components during testing, may I know how to define or find 'insignificant components'? Any pattern there? Will there be a mess attention map as indicator or something else?

3. As far as my understanding, currently the trained model supports one attribute editing and we need to train different models for different attributes. Is it right? If yes, it is possible to extend it to a multi-domain transformation model?

As some details are ignored in the main paper, I guess there should be a supplementary material. Could you please share this with me, so that I can understand your nice work better.

Looking forward to hearing from you!
Best regards.

**Comment by yingcong** [comment]
Hi @RegisWu. Thanks for interested in this work!
1. The loss function is L2 between the network output feature and the ground truth Vs. 
2. There are some components that attend on background, or wrong location (like Fig.4 (b)). Developers should select them and manually by seeing 1. the attention map and 2. the result after disabling the corresponding component. 
3. Yes, the current model supports one attribute editing, and it seems not so intuitive to extend it to handle multi-domain translation. This model is quite simple at this moment, and it just attempt to show that ONE high-level semantic attribute can be decomposed, and e need to train different model for different attributes. 
4. The supplementary file can be found here: http://openaccess.thecvf.com/content_CVPR_2019/supplemental/Chen_Semantic_Component_Decomposition_CVPR_2019_supplemental.pdf

**Comment by RegisWu** [comment]
Got it! Thanks!
